org.fooyin.fooyin.yaml

```diff
@@ -0,0 +1,106 @@
+app-id: org.fooyin.fooyin
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: fooyin
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '23.08'
+    add-ld-path: .
+finish-args:
+  - '--device=dri'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--filesystem=/run/media'
+  - '--own-name=org.mpris.MediaPlayer2.fooyin'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--system-talk-name=org.freedesktop.UDisks2'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.gtk.Notifications'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
```

Comment:
I don't see it having tray functionality?

Reply:
>  '--system-talk-name=org.freedesktop.UDisks2'

is needed, otherwise it throws an error at startup regarding udisk.

Removing notification stuff.

Reply:
I'm talking about `org.kde.StatusNotifierWatcher`

Reply:
I see it was removed in https://github.com/flathub/flathub/pull/5261/commits/e767076c77290bbe179cb7ce98e2b7837d64dcb5 resolving.